src/Test/StateMachine/Internal/Sequential.hs

```diff
@@ -150,7 +150,7 @@ liftSem
 liftSem sem cmd iref = do
 
   env <- get
-  let cmd' = ifmap @_ @_ @_ @_ @refs (\s i -> env IxM.! (s, i)) cmd
+  let cmd' = ifmap @_ @_ @_ @_ @refs (curry ((IxM.!) env)) cmd
```

Comment:
Me neither, but then we need to figure out how to disable these kind of hints...